docker/yoti/data/configuration.json

```
@@ -0,0 +1,142 @@
+    }
+  },
+  "capture": {
+    "required_resources": [
+      {
+        "type": "ID_DOCUMENT",
+        "id": "d430440c-5c6d-402c-9855-3aebc8145a94",
+        "state": "REQUIRED",
+        "allowed_sources": [
+          {
+            "type": "IBV"
+          }
+        ],
+        "requested_tasks": [],
+        "ibv_client_assessments": [
+          {
+            "type": "IBV_VISUAL_REVIEW_CHECK",
+            "state": "REQUIRED"
+          },
+          {
+            "type": "DOCUMENT_SCHEME_VALIDITY_CHECK",
+            "state": "REQUIRED",
+            "scheme": "UK_DBS"
+          },
+          {
+            "type": "PROFILE_DOCUMENT_MATCH",
+            "state": "REQUIRED"
+          }
+        ],
+        "supported_countries": [
+          {
+            "code": "GBR",
+            "supported_documents": [
+              {
+                "type": "PASSPORT"
+              }
+            ]
+          }
+        ],
+        "allowed_capture_methods": "CAMERA",
+        "attempts_remaining": {
+          "RECLASSIFICATION": 2,
+          "GENERIC": 2
+        }
+      },
+      {
+        "type": "ID_DOCUMENT",
+        "id": "d33f1c38-2f4a-4d10-a22b-971ac63132ce",
+        "state": "REQUIRED",
+        "allowed_sources": [
+          {
+            "type": "IBV"
+          }
+        ],
+        "requested_tasks": [],
+        "ibv_client_assessments": [
+          {
+            "type": "IBV_VISUAL_REVIEW_CHECK",
+            "state": "REQUIRED"
+          },
+          {
+            "type": "DOCUMENT_SCHEME_VALIDITY_CHECK",
+            "state": "REQUIRED",
+            "scheme": "UK_DBS"
+          },
+          {
+            "type": "PROFILE_DOCUMENT_MATCH",
+            "state": "REQUIRED"
+          }
+        ],
+        "supported_countries": [
+          {
+            "code": "GBR",
+            "supported_documents": [
+              {
+                "type": "DRIVING_LICENCE"
+              }
+            ]
+          }
+        ],
+        "allowed_capture_methods": "CAMERA",
+        "attempts_remaining": {
+          "RECLASSIFICATION": 2,
+          "GENERIC": 2
+        }
+      },
+      {
+        "type": "SUPPLEMENTARY_DOCUMENT",
+        "id": "b23048de-0927-4a42-bd38-762d904d9fd7",
+        "state": "REQUIRED",
+        "allowed_sources": [
+          {
+            "type": "IBV"
+          }
+        ],
+        "requested_tasks": [],
+        "ibv_client_assessments": [
+          {
+            "type": "IBV_VISUAL_REVIEW_CHECK",
+            "state": "REQUIRED"
+          },
+          {
+            "type": "DOCUMENT_SCHEME_VALIDITY_CHECK",
+            "state": "REQUIRED",
+            "scheme": "UK_DBS"
+          },
+          {
+            "type": "PROFILE_DOCUMENT_MATCH",
+            "state": "REQUIRED"
+          }
+        ],
+        "document_types": ["UTILITY_BILL"],
+        "country_codes": ["GBR"],
+        "objective": {
+          "type": "UK_DBS"
+        }
+      }
```

In our usage there'll only ever be one required resource (the ID) so I suggest we only use one here to be a bit more reflective of what a real response will look like.
```suggestion
      }
```